src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -147,4 +147,12 @@ ALTER TABLE pg_dist_node ADD COLUMN metadatasynced BOOLEAN DEFAULT FALSE;
 COMMENT ON COLUMN pg_dist_node.metadatasynced IS
     'indicates whether the node has the most recent metadata';
 
+CREATE FUNCTION worker_apply_sequence_command(create_sequence_command text,
```

Comment:
So, we'll have two functions on the workers: `worker_apply_sequence_command (text)` and `worker_apply_sequence_command (text, regtype)`. Is that intentional?

Reply:
It was intentional in the sense that I was thinking of rolling upgrade scenarios where workers are ahead of the coordinator.